warehouse/query-core/src/main/java/datawave/query/jexl/lookups/ShardIndexQueryTableStaticMethods.java

```diff
@@ -381,18 +381,18 @@ public static ScannerSession configureTermMatchOnly(ShardQueryConfiguration conf
         
         bs.setRanges(ranges);
         
-        SessionOptions options = new SessionOptions();
-        
-        IteratorSetting setting = configureDateRangeIterator(config);
-        options.addScanIterator(setting);
-        
-        setting = configureGlobalIndexTermMatchingIterator(config, literals, patterns, reverseIndex, limitToUniqueTerms);
-        if (setting != null) {
+        try (SessionOptions options = new SessionOptions()) {
```

Comment:
Interesting that SessionOptions.close is a noop.  I guess there's no harm in closing it even though it doesn't do anything.